examples/benchmark/tls_bench.c

```diff
@@ -291,9 +291,14 @@ static struct group_info groups[] = {
     { WOLFSSL_KYBER_LEVEL1, "KYBER_LEVEL1" },
     { WOLFSSL_KYBER_LEVEL3, "KYBER_LEVEL3" },
     { WOLFSSL_KYBER_LEVEL5, "KYBER_LEVEL5" },
-    { WOLFSSL_P256_KYBER_LEVEL1, "P256_KYBER_LEVEL1" },
-    { WOLFSSL_P384_KYBER_LEVEL3, "P384_KYBER_LEVEL3" },
-    { WOLFSSL_P521_KYBER_LEVEL5, "P521_KYBER_LEVEL5" },
+    { WOLFSSL_P256_KYBER_LEVEL1, "P256_KYBER_LEVEL1"     },
+    { WOLFSSL_P384_KYBER_LEVEL3, "P384_KYBER_LEVEL3"     },
+    { WOLFSSL_P256_KYBER_LEVEL3, "P256_KYBER_LEVEL3"     },
+    { WOLFSSL_P521_KYBER_LEVEL5, "P521_KYBER_LEVEL5"     },
+    { WOLFSSL_P384_KYBER_LEVEL5, "P384_KYBER_LEVEL5"     },
+    { WOLFSSL_X25519_KYBER_LEVEL1, "X25519_KYBER_LEVEL1" },
+    { WOLFSSL_X448_KYBER_LEVEL3, "X448_KYBER_LEVEL3"     },
+    { WOLFSSL_X25519_KYBER_LEVEL3, "X25519_KYBER_LEVEL3" },
```

Comment:
Its interesting.  

You are adding : 
- WOLFSSL_X25519_KYBER_LEVEL1
- WOLFSSL_X448_KYBER_LEVEL3
- WOLFSSL_X25519_KYBER_LEVEL3
- WOLFSSL_P256_KYBER_LEVEL3
- WOLFSSL_P384_KYBER_LEVEL5

Can you let us know why you are adding these?  I suppose they are to interop with other places, but can you let us know specifically (ie which ones are mozilla and which ones are liboqs)

Reply:
I am currently investigating the real-world performance impact of various PQC deployments in different embedded context (for my PhD). As hybrid constellations using `X25519` and `X448` are pretty popular in other projects, I wanted to test them, too. That's why I started this work in the first place. 

The compatibility with the web browsers is based on the use case to access a web-based management interface of an embedded device and to make that PQC secure. For that, `WOLFSSL_X25519_KYBER_LEVEL3` is the relevant hybrid constellation (that is what browser vendors are currently settled on).

The other constellations are based on what OQS has defined. I added them (besides my internal performance tests) for general interoperability with OQS and because it wasn't much overhead anyway.